# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/upower/hotfix-gudev-version.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

Eudev is currently missing some functions required to build gudev 238
so we kept it to 237 in T2 trunk at the moment which seems to work
without issues.

  - NoTag <notag@maneto.fr>

--- upower-v1.90.2/meson.build.vanilla	2023-08-05 19:30:11.544798934 +0200
+++ upower-v1.90.2/meson.build	2023-08-05 19:30:20.982525496 +0200
@@ -69,7 +69,7 @@
 
 
 if os_backend == 'linux'
-  gudev_dep = dependency('gudev-1.0', version: '>= 238')
+  gudev_dep = dependency('gudev-1.0', version: '>= 237')
   idevice_dep = dependency('libimobiledevice-1.0',
                            version : '>= 0.9.7',
                            required : get_option('idevice'))
